mypy/semanal.py

```diff
@@ -159,6 +159,8 @@ class SemanticAnalyzer(NodeVisitor):
     modules = None  # type: Dict[str, MypyFile]
     # Global name space for current module
     globals = None  # type: SymbolTable
+    # Globals that were mentioned in __all__ and should be module_public.
+    global_exports = set()
```

Comment:
This variable is unused. Maybe you changed your mind and named it all_exports later?